src/freedv_api.c

```diff
@@ -431,6 +438,17 @@ void freedv_datatx (struct freedv *f, short mod_out[]) {
     }
 }
 
+void freedv_datacomptx(struct freedv *f, COMP mod_out[])
+{
```

Comment:
Is this function tested in this PR?

Reply:
No unittest yet, but I do use it with freedv_gui

Reply:
I added a usecomplex flag to freedv_data_tx and freedv_data_rx and created testcases for them